modules/intentIqIdSystem.md

```diff
@@ -47,6 +47,7 @@ Please find below list of parameters that could be used in configuring Intent IQ
 | params.domainName              | Optional | String   | Specifies the domain of the page in which the IntentIQ object is currently running and serving the impression. This domain will be used later in the revenue reporting breakdown by domain. For example, cnn.com. It identifies the primary source of requests to the IntentIQ servers, even within nested web pages.                     | `"currentDomain.com"`                         |
 | params.gamObjectReference      | Optional | Object   | This is a reference to the Google Ad Manager (GAM) object, which will be used to set targeting. If this parameter is not provided, the group reporting will not be configured.                                                                                                                                                            | `googletag`                                   |
 | params.gamParameterName        | Optional | String   | The name of the targeting parameter that will be used to pass the group. If not specified, the default value is `intent_iq_group`.                                                                                                                                                                                                        | `"intent_iq_group"`                           |
+| params.gamPredictReporting        | Optional | Boolean   | This variable controls whether the GAM prediction logic is enabled or disabled.                                                                                                                                                                                                        | `false`                           |
```

Comment:
@DimaIntentIQ it seems you have configuration of the analytics module in the id module configuration